picard.spec

```diff
@@ -28,9 +29,17 @@ def get_locale_messages():
     return data_files
 
 
-
 block_cipher = None
 data_files = get_locale_messages()
+os_name = platform.system()
```

Comment:
Just to be picky i would just write things this way:

```python
if platform.system() == 'Windows':
  fpcalc_name = 'fpcalc.exe'
else:
  fpcalc_name = 'fpcalc'
```

I don't see the point of using string concatenation to "build" the file name, `os_name` would be useful if used more than once, which isn't the case here.

Reply:
It is used in the OSX pr again, so I thought I'd just include it here.